app/routes/your-claims/view-claim.js

```diff
@@ -148,11 +150,17 @@ module.exports = function (router) {
     }
 
     getClaimDocumentFilePath(req.params.claimDocumentId)
-      .then(function (result) {
+      .then(async function (result) {
```

Comment:
similarly here.. could potentially await this?

Reply:
Probably. I'm trying to change as little as possible to get stuff working :)